theme.json

```diff
@@ -0,0 +1,413 @@
+{
+  "$schema": "https://schemas.wp.org/trunk/theme.json",
+  "version": 2,
+  "settings": {
+    "appearanceTools": true,
+    "useRootPaddingAwareAlignments": true,
+    "border": {
+      "color": false,
+      "radius": false,
+      "style": false,
+      "width": false
+    },
+    "color": {
+      "text": false,
+      "background": false,
+      "link": false,
+      "custom": false,
+      "defaultPalette": false,
+      "defaultGradients": false,
+      "defaultDuotone": false,
+      "customGradient": false,
+      "customDuotone": false,
+      "palette": [
+        {
+          "slug": "text",
+          "color": "var(--wp--custom--text--secondary)",
+          "name": "text"
+        },
+        {
+          "slug": "heading",
+          "color": "var(--wp--custom--text--primary)",
+          "name": "heading"
+        },
+        {
+          "slug": "background",
+          "color": "var(--wp--custom--background--primary)",
+          "name": "background"
+        }
+      ],
+      "gradients": []
+    },
+    "custom": {
+      "body": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "16px"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 400,
+          "line-height": "140%",
+          "font-size": "18px"
+        }
+      },
+      "heading": {
+        "1": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.07rem, calc(1.02rem + 0.25vw), 1.28rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "2": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.14rem, calc(1.04rem + 0.47vw), 1.53rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "3": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.30rem, calc(1.08rem + 1.09vw), 2.20rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "4": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.38rem, calc(1.08rem + 1.52vw), 2.64rem)",
+          "letter-spacing": "-0.01em"
+        },
+        "5": {
+          "font-family": "Aktiv Grotesk VF",
+          "font-weight": 500,
+          "line-height": "clamp(1.1em, calc(1.25em + -0.4vw), 1.2em)",
+          "font-size": "clamp(1.57rem, calc(1.04rem + 2.69vw), 3.81rem)",
+          "letter-spacing": "-0.01em"
+        }
+      },
+      "space": {
```

Comment:
I think we'll have to make a call on spacing. Technically, we should have a set of primitive spacings and then apply them to each component which would get it's own spacing tokens. So we would start with space-1 space-2 space-3 etc. and then apply them like this: button-padding = space-1 form-gap = space-2 etc. This makes everything nice and clear and makes extending the system a bit more straightforward but ... it make a lot of tokens ... which is less than ideal if you're trying to keep the set pretty tight. The way I dealt with this in valet was to make semantic tokens for unique things (I think it was just global padding) and then basically pull the primitives forward into the theme. So that's where you get the set of static and responsive spacing tokens in the valet theme.json. There are a bunch of them but they are one of the most useful features. Without them a dev would have to use arbitrary spaces in all the block styles. It doesn't guide the user on how to apply the spacing values — which is what a real semantic token set would do — but in practice it works pretty well and keeps things a lot more consistent just by limiting the number of possible options. I would suggest including some static and responsive spacing tokens here as part of a basic theme. It needs documentation but it's darn useful. I can help write that if needed.

Reply:
Just so I understand this correctly, you are suggesting to change this into something like:

```
      "space": {
        "responsive": {
          "1": "clamp(1.00rem, calc(0.40rem + 3.01vw), 3.50rem)"
        },
        "static": {
          "1": "clamp(1.50rem, calc(1.38rem + 0.60vw), 2.00rem)"
        }
      },
```

Reply:
I can see the benefit of that, especially when I look at valet as well as CDT. I would want to keep this set small and leave it at just 1 entry for each however. If a dev wants to add more they can always do that down the line